lib/db/models/Node.ts

```diff
@@ -7,6 +7,16 @@ export default (sequelize: Sequelize.Sequelize, DataTypes: Sequelize.DataTypes)
     id: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
     nodePubKey: { type: DataTypes.STRING, unique: true, allowNull: false },
     addressesText: { type: Sequelize.TEXT, allowNull: false },
+    lastAddressText: { type: Sequelize.TEXT, allowNull: false },
+    lastAddress: {
+      type: Sequelize.VIRTUAL,
```

Comment:
Can you move these new lines down so they're below `addresses` - it'd be nice to keep `addressesText` and `addresses` adjacent.

Reply:
done :)